tapeagents/nodes.py

```diff
@@ -52,7 +52,16 @@ def prepare_tape(self, tape: Tape) -> Tape:
         return tape.model_copy(update=dict(steps=steps_without_control_flow))
 
     def make_llm_output(self, agent: Any, tape: Tape, index: int) -> LLMOutput:
-        return LLMOutput(role="assistant", content=tape.steps[index].llm_view())
+        steps = []
```

Comment:
can you add a docstring that explains that what we do here is a heuristic, and how it works?